cli/lib/output-filter.ts

```diff
@@ -58,6 +58,14 @@ export function removeHydraTriples({ subject, predicate, object }: Quad): boolea
   return !(isHydraTerm(subject) || isHydraTerm(predicate) || isHydraTerm(object))
 }
 
+function isOriginalValueTerm(term: Term): boolean {
+  return term.termType === 'NamedNode' && term.value.endsWith('#_original')
+}
+
+export function removeMappingsOriginalValues({ subject, predicate, object }: Quad): boolean {
+  return !(isOriginalValueTerm(subject) || isOriginalValueTerm(predicate) || isOriginalValueTerm(object))
```

Comment:
Why not just check predicates? When will there be a `#_original` as subject or object?